src/main/java/lotto/model/LottoGenerator.java

```diff
@@ -2,61 +2,105 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.Collections;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import lotto.code.ErrorCode;
+import lotto.exception.LottoException;
+import lotto.util.RandomUtil;
+
 public class LottoGenerator {
-	private final Money money;
-	private final List<String> inputNumberList;
+	private static final String IS_NUMBER_REGEX = "[1-9]";
+	private static final int ONLY_RANDOM_NUMBERS = 0;
 
-	private LottoGenerator(Money money, List<String> inputNumberList) {
-		this.money = money;
-		this.inputNumberList = Collections.unmodifiableList(inputNumberList);
+	private LottoGenerator() {
 	}
 
-	public static LottoGenerator from(Money money) {
-		return new LottoGenerator(money, new ArrayList<>());
+	public static LottoGenerator getInstance() {
+		return new LottoGenerator();
 	}
 
-	public static LottoGenerator of(Money money, List<String> inputNumberList) {
-		return new LottoGenerator(money, inputNumberList);
+	private static void validInputSize(String input, int inputNumberListSize) {
```

Comment:
객체로 사용도록 정의해주셨다면 정적 메서드가 아니어도 괜찮을 것 같아요 :)
내부에 static 메서드들을 제거해주세요 :)

```suggestion
	private void validInputSize(String input, int inputNumberListSize) {
```